discovery/discovery.go

```diff
@@ -114,7 +115,7 @@ func (o *orchestratorPool) GetOrchestrators(numOrchestrators int, suspender comm
 			return
 		}
 		if err != nil && !errors.Is(err, context.Canceled) {
-			glog.Error(err)
+			clog.Errorf(ctx, "err=%q", err)
```

Comment:
Hmm, isn't `Infof`(err) a good candidate here? Either way came to my mind that `Infofe` will be logging errors as info, not as errors, right? Sth we'd like to change somehow?

Also, while at it, mind adding any text here to say something about what is this error?

Reply:
because `ctx` is captured in the closure, and will be the same for subsequent calls to function that is returned from `createRTMPStreamIDHandler`